docs/source/python/models/detection_model.md

```diff
@@ -34,11 +31,9 @@ model = SSD.create_model("model.xml")
 # Forward pass
 predictions = model(image)
 
-# Iterate over the segmented objects
-for pred_obj in predictions.objects:
-    pred_score = pred_obj.score
-    label_id = pred_obj.id
-    bbox = [pred_obj.xmin, pred_obj.ymin, pred_obj.xmax, pred_obj.ymax]
+# Iterate over detection result
+for box, score, label in zip(predictions.boxes, predictions.scores, predictions.labels):
+    print(f"Box: {box}, Score: {score}, Label: {label}")
```

Comment:
Does it make sense to add a string label here?

Reply:
Updated!